packages/core/src/types.ts

```diff
@@ -104,6 +104,7 @@ export enum SetOp {
 export enum SortDirection {
 	Ascending = 'asc',
 	Descending = 'desc',
+	Unsorted = 'unsorted',
```

Comment:
Could this just be represented by an undefined SortDirection in state rather than an enum value?